fv3core/fv3core/stencils/divergence_damping.py

```diff
@@ -407,16 +402,15 @@ def __call__(
             u (in):
             v (in):
             va (in):
-            u_contra_dyc (out):
-            v_contra_dxc (out):
+            v_contra_dxc (out): wk converted from a grid to b grid and damped
```

Comment:
While you're at it, does it make sense to rename the arguments in damping to match (`vort` to `v_contra_dxc`)?  Or does this variable definition gets changed?

Reply:
At this point I believe `vort` is actually more accurate, I'm leaving renaming this out of this PR though. I believe this is vorticity on the b-grid.

It's taken a long time to figure this out since "vort" is used for so many other things as well.